content/administration/upgrade.rst

```diff
@@ -1,12 +1,252 @@
+script.
+
+If you do not find any discrepancies, you can move on to the upgrade of your production database.
+
+.. important::
+   A test database is only intended for testing and remains completely unrelated to your present or
+   future production database. Any data you add, or changes you make, will not be reflected in your
+   upgraded production database.
+
+.. note::
+   Test databases are neutered and features are disabled to prevent them from having an impact on
+   the production database:
+
+   #. The serial number of the database is modified (to prevent it from sending information as if it
+      was the production database).
+   #. The :ref:`base URL of the database <domain-name/web-base-url>` is reset to
+      ``http://localhost:8069`` and the email domain to ``localhost``.
+   #. Scheduled actions are disabled (the calendar synchronization, the bank statement
+      synchronization, the planned automated actions, the fetching of incoming mail servers, etc.).
+   #. Outgoing mail servers are disabled by archiving the existing ones and adding a
+      fake/non-working one.
+   #. Payment providers and delivery carriers are reset to test environment.
+   #. Accounting localization Electronic Data Interchange (EDI) services are disabled.
+   #. A system parameter is set to tell the database has been neutered.
+
+.. _upgrade/test-db-request:
+
+Request a test database
+=======================
+
+Follow the instructions available per hosting type on the `website form
+<https://upgrade.odoo.com>`_. and select *Testing* purpose.
+
+.. image:: upgrade/test-purpose.png
+   :align: center
+   :alt: Selection of the "Testing" purpose in the upgrade form on Odoo
+
+.. _upgrade/test-guidance:
+
+Test guidance
+=============
+
+Every business and organization has its own operational needs and has to test its specific Odoo
+database individually. We recommend you look at `the test scenario
+<https://docs.google.com/document/d/1ypNs7JKPOsjNbKpdiKFH7Al6g6whZ9jr7f7duAQ5E1w/>`_ for further
+information.
+
+.. todo:: change link "test scenario" once the related doc is published
+
+.. _upgrade/test-assistance:
+
+Assistance
+----------
+
+If you encounter an issue in the **test database**, please get in touch with Odoo Upgrade Support
+via the `Odoo Support page <https://www.odoo.com/help>`_.
+
+Under the *Ticket Description* section, select *An issue related to my upgrade* ticket type.
+
+   .. image:: upgrade/test-assistance.png
+      :align: center
+      :alt: Selection of "An issue related to my upgrade" as Ticket Type in the support form on Odoo
+
+   .. warning::
+      If you choose another *Ticket Description* type, the request will be redirected to another
+      team. This will slow down the processing and response time.
+
+Please provide as much detail as you can (i.e., videos and screenshots to illustrate your issue).
+This will avoid clarifying questions and speed up the resolution process significantly.
```

Comment:
```suggestion
This will allow clarifying questions and speed up the resolution process significantly.
```
```suggestion
This will avoid clarifying questions and speed up the resolution process significantly.
```